internal/iam/command_rbac_role_binding.go

```diff
@@ -54,6 +55,8 @@ var (
 
 	literalPatternType  = "LITERAL"
 	prefixedPatternType = "PREFIXED"
+
+	nonUserPrincipalTypes = []string{presource.ServiceAccount, presource.IdentityPool, presource.SsoGroupMapping}
```

Comment:
Let's define this in the function it's used in? No need for it to be global in this package